### ISSUE TITLE
Include short description in Instance Manager while browsing/adding Mods

### ISSUE THREAD

**Issue by recon88** [killpowa/Twickt-Launcher#1277]
I really would love to have a short description of the mods you can scroll through in the instance manager.
Currently you have to click on every single mod if you want to know what it does.
Pretty sure the API returns a 'summary' field that can be used for that.

Additionally a couple more details like download count, update date/time and MC version like shown below would be nice as well.

GDLauncher:
https://i.imgur.com/UMRF8sv.png

Comparison:
https://i.imgur.com/buPiyOS.png

**Comment by CritLoren** [comment]
Yeah that'd be nice, same for modlists. Just having the title isn't enough to get a good read on a list, and one can't be expected to click more info on each card.
